bundles/org.openhab.binding.astro/README.md

```diff
@@ -196,7 +196,7 @@ Number:Intensity Total_Radiation    "Radiation [%.2f %unit%]"                 {
 Number:Intensity Diffuse_Radiation  "Diffuse Radiation [%.2f %unit%]"         { channel="astro:sun:home:radiation#diffuse" }
 ```
 
-Events:
+Events (Note: offset/latest on #start or #end are independent from offset/latest on the #event, and only #event can be used in rules triggers like below):
```

Comment:
"earliest" is not mentioned. Did you mismatch "offset" and "earliest" ?

"offset", "earliest" and "latest" are channel options I believe, so yes of course they can't be use as trigger in a rule.
Maybe you should rather add your explanations in the part of the README defining these parameters ?

Reply:
I would suggest to move it to line 97

Reply:
> "earliest" is not mentioned. Did you mismatch "offset" and "earliest" ?

No, I simply didn't want to enumerate everything again.
 
> "offset", "earliest" and "latest" are channel options I believe, so yes of course they can't be use as trigger in a rule. Maybe you should rather add your explanations in the part of the README defining these parameters ?

You say "of course", but this is far from obvious for a normal user like me, I had to debug and research this for several evenings when the rules didn't trigger as I thought they should trigger, even though the sunset item was showing the offset correctly.

I've moved this up to line 97 as suggested.